docs/modules/ROOT/pages/mathematical-operators.adoc

```diff
@@ -0,0 +1,93 @@
+[[mathematical-operators]]
+= Mathematical operators
+
+Mathematical operators are a handy way to update numerical fields in a single roundtrip to the DB.
+
+Specfic operators are available on different numerical types (`Int`, `Float`, xref::type-definitions/types.adoc#type-definitions-types-bigint[`BigInt`]).
+
+We chose to keep operators separated by types as some operators are not available to Natural numbers.
+
+== Supported locations
+Mathematical operators are supported in Update Mutations within these entities:
+
+* `Node`
+* `Nested Nodes`
+* `Relationship properties`
+* `Interfaces`
+
+== Int and BigInt operators
+For `Int` and `BigInt` types, the following operators are available:
+
+* `_DECREMENT`
+* `_INCREMENT`
+
+== Float operators
+For `Float` type, the following operators are available:
+
+* `_ADD`
+* `_SUBTRACT`
+* `_MULTIPLY`
+* `_DIVIDE`
+
+== The social platform schema
+In this section, we use the following GraphQL schema:
+
+[source, graphql, indent=0]
+----
+type Video {
+  id: ID @id
+  views: Int
+  ownedBy: User @relationship(type: "OWN_VIDEO", properties: "OwnVideo", direction: IN)
+}
+type User {
+  id: ID @id
+  ownVideo: [Video!]! @relationship(type: "OWN_VIDEO", properties: "OwnVideo", direction: OUT)
+}
+interface OwnVideo @relationshipProperties {
+  revenue: Float
+}
+----
+
+
+=== Simple increment operation
+Let's say that a user visualizes a video, so we want to increment `viewersCount` for that video by 1.
```

Comment:
```suggestion
Let's say that a user views a video, so we want to increment `viewersCount` for that video by 1.
```

Unless you specifically want to use the word `visualizes` for some reason, I think `views` works better here.